changelog.d/568.change.rst

```diff
@@ -0,0 +1 @@
+The value passed to ``@attr.ib(repr=…)`` can now be either a boolean (as before) or a custom callable, which makes it possible to influence how the attribute value is formatted by the generated ``__repr__()`` method.
```

Comment:
I find this a big vague if you don't know what this is about.

How about this:

```rst
The value passed to ``@attr.ib(repr=…)`` can now be either a boolean (as before) or a callable.
That callable must return a string and is then used for formatting the attribute by the generated ``__repr__()`` method.
```

Reply:
no strong opinion; applied your changes as-is: 72046cb9c79c0bdbad9ed29a8f5a9ec80b1d277b